crates/nargo/tests/run_examples.rs

```diff
@@ -8,7 +8,7 @@ fn run_examples() {
     let examples_dir = format!("{}/../../examples", env!("CARGO_MANIFEST_DIR"));
 
     let paths = std::fs::read_dir(&examples_dir)
-        .expect(&format!("Could not read from directory {examples_dir}"));
+        .expect(&format!("Could not read from directory {}", examples_dir));
```

Comment:
Did you ever figure out what this error was caused by? I looked it up to see if the `"... {variablename}"` syntax was new but it has been supported since January 2020: https://github.com/rust-lang/rfcs/pull/2795

Reply:
No, but writing like this make it work for me.